tests/FunctionalTestCase.php

```diff
@@ -484,6 +484,13 @@ protected function isShardedClusterUsingReplicasets()
         return preg_match('@^.*/.*:\d+@', $document['host']);
     }
 
+    protected function skipIfServerVersionNotSupported(string $operator, string $version, ?string $message = null): void
```

Comment:
How do you feel about naming this `skipIfServerVersion()? That would both make the LOC shorter and allow it to read like a natural sentence. For example:

```php
$this->skipIfServerVersion('<', '7.0.0', '...');
```

Reply:
Less words is more correct 🥂